clang/lib/Basic/Targets/AArch64.cpp

```diff
@@ -286,7 +286,6 @@ void AArch64TargetInfo::getTargetDefinesARMV84A(const LangOptions &Opts,
 void AArch64TargetInfo::getTargetDefinesARMV85A(const LangOptions &Opts,
                                                 MacroBuilder &Builder) const {
   Builder.defineMacro("__ARM_FEATURE_FRINT", "1");
-  Builder.defineMacro("__ARM_FEATURE_BTI", "1");
```

Comment:
Don't know if this should change. It's part of the 8.5 feature set after all.

Reply:
Generally speaking, we should gate things by specific feature rather than architecture, so I think this change is correct regardless of what we do with these specific chips.

Reply:
may as well do `FRINT` too while you're here?

Reply:
Maybe in a separate PR.